django/core/management/commands/test.py

```diff
@@ -48,6 +48,8 @@ def add_arguments(self, parser):
             help='Overrides the default address where the live server (used '
                  'with LiveServerTestCase) is expected to run from. The '
                  'default value is localhost:8081-8179.'),
+        parser.add_argument('--tags', action='store', dest='tags',
+            help='Tells Django to only run tests having specified tags. You can specify multiple tags with a comma.'),
```

Comment:
Do we want to specify multiple tags as multiple `--tag=` instead of a comma separated list instead?